src/components/Logo.js

```diff
@@ -14,7 +14,11 @@ const Logo = ({ src, style = {}, height, type = 'ORGANIZATION', website }) => {
     backgroundStyle.minWidth = parseInt(height, 10) / 2;
   }
   if (!src) {
-    backgroundStyle.backgroundImage = `url(${defaultImage[type]})`;
+    if (defaultImage[type]) {
+      backgroundStyle.backgroundImage = `url(${defaultImage[type]})`;
+    } else {
+      console.warn(`No defaultImage for type '${type}'`);
```

Comment:
Do we want to keep this in production?

Reply:
@HipsterBrown I didn't want to only avoid the problem, I wanted to give some visibility to it.

What do you think we should do?

Reply:
I would probably make it a `debug` statement, otherwise it would probably be something we report to a system like Rollbar. Is it really a warning if we don't have a default image for a collective type?